webknossos/webknossos/dataset/_utils/pims_images.py

```diff
@@ -0,0 +1,243 @@
+import warnings
+from contextlib import contextmanager, nullcontext
+from os import PathLike
+from typing import Iterator, List, Optional, Sequence, Tuple, TypeVar, Union, cast
+from urllib.error import HTTPError
+
+import numpy as np
+import pims
+
+from webknossos.dataset.mag_view import MagView
+from webknossos.geometry.vec3_int import Vec3Int
+
+
+class PimsImages:
+    def __init__(
+        self,
+        images: Union[str, "pims.FramesSequence", List[Union[str, PathLike]]],
+        timepoint: Optional[int],
+        swap_xy: bool,
+        flip_x: bool,
+        flip_y: bool,
+        flip_z: bool,
+        use_bioformats: bool,
+    ) -> None:
+        self._original_images = images
+        del images
```

Comment:
why is it necessary to `del` it? it's still available via `self._original_images` which is why I would have thought that this wouldn't be needed.